.github/workflows/ci.yaml

```diff
@@ -44,6 +44,15 @@ jobs:
       - name: E2E Tests
         run: npm run e2e
 
+      - name: Upload VRT report
+        if: always()
```

Comment:
Just curious, is this line necessary?

Reply:
I included it because it was in the sample config I found in the docs, my guess is that it causes the upload to run even if the previous test step failed.